docs/source/federation/errors.md

```diff
@@ -34,6 +34,12 @@ Apollo Federation implements a strict composition model. When building a gateway
 - `REQUIRES_FIELDS_MISSING_ON_BASE`: The fields arg in `@requires` can only reference fields on the base type.
 - `REQUIRES_USED_ON_BASE`: The requires directive may not be used on fields defined on a base type.
 
+### Custom directives
+
+- `EXECUTABLE_DIRECTIVES_ONLY`: Custom directives may only be [`ExecutableDirective`s](https://graphql.github.io/graphql-spec/June2018/#ExecutableDirectiveLocation). The locations which a directive implements must be a subset of: `QUERY`, `MUTATION`, `SUBSCRIPTION`, `FIELD`, `FRAGMENT_DEFINITION`, `FRAGMENT_SPREAD`, and `INLINE_FRAGMENT`.
+- `EXECUTABLE_DIRECTIVES_IN_ALL_SERVICES`: Custom directives must be implemented across all services. It's acceptable to implement a directive as a no-op within a particular service, but it must still be defined.
+- `EXECUTABLE_DIRECTIVES_IDENTICAL`: Custom directives must be implemented identically from one service to the next. This means that arguments and their respective types, as well as the directives locations, must all be identical within every service.
```

Comment:
"from one service to the next" -> "across all services"
"directives locations" -> "directive's locations"